lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -770,7 +770,7 @@
         version_added: "2.8"
     force_migrate:
         description:
-            - "If I(true), the VM will migrate even if it is defined as non-migratable."
+            - "If I(true), the VM will migrate when vm is user-migratable."
```

Comment:
The three migration modes referred to in the bug report are defined in the `placement_policy` option. Giving users a pointer to that option will help them understand all possible settings. 

```suggestion
            - If I(true), the VM will migrate when I(placement_policy=user-migratable) but not when I(placement_policy=pinned).
```